docs/reference/contrib/routablepage.md

```diff
@@ -137,10 +137,10 @@ def next_event(self, request):
     'year/2015/'
 ```
 
-This method only returns the part of the URL within the page. To get the full URL, you must append it to the values of either the {attr}`~wagtail.models.Page.url` or the {attr}`~wagtail.models.Page.full_url` attribute on your page:
+This method only returns the part of the URL within the page. To get the full URL, you must append it to the values of either the {meth}`~wagtail.models.Page.get_url` method or the {attr}`~wagtail.models.Page.full_url` attribute on your page:
 
 ```python
->>> event_page.url + event_page.reverse_subpage('events_for_year', args=(2015, ))
+>>> event_page.get_url() + event_page.reverse_subpage('events_for_year', args=(2015, ))
```

Comment:
q: I am curious as to why change to the method here, but leave the `full_url` attribute below? Both `Page.url` and `Page.full_url` are defined as properties via `url = property(get_url)` and `full_url = property(get_full_url)` respectively

Reply:
Not sure if it's related but `get_url` is documented while `url` isn't, as of 8d786c35349f097b1c9c8cdc5c36f81cb68386ec.

Reply:
huh, I missed that commit.